sql/dolphinscheduler-postgre.sql

```diff
@@ -752,7 +752,7 @@ ALTER TABLE t_ds_worker_server ALTER COLUMN id SET DEFAULT NEXTVAL('t_ds_worker_
 
 
 -- Records of t_ds_user?user : admin , password : dolphinscheduler123
-INSERT INTO t_ds_user(user_name,user_password,user_type,email,phone,tenant_id,state,create_time,update_time) VALUES ('admin', '7ad2410b2f4c074479a8937a28a22b8f', '0', 'xxx@qq.com', 'xx', '0', 1, '2018-03-27 15:48:50', '2018-10-24 17:40:22');
+INSERT INTO t_ds_user(user_name,user_password,user_type,email,phone,tenant_id,state,create_time,update_time) VALUES ('admin', '7ad2410b2f4c074479a8937a28a22b8f', '0', 'xxx@qq.com', '', '0', 1, '2018-03-27 15:48:50', '2018-10-24 17:40:22');
```

Comment:
Very good job,
Sorry about I didn't notice that these two places before, and these tow places also need to be changed.
![image](https://user-images.githubusercontent.com/29545877/88470854-533d5980-cf34-11ea-80c2-87ec81da2060.png)
![image](https://user-images.githubusercontent.com/29545877/88470862-7962f980-cf34-11ea-892e-f74d84013f6e.png)

Reply:
I have deleted and submitted.